content-repo/extra-docs/articles/IAM-premium-pack-readme.md

```diff
@@ -207,6 +219,7 @@ The following is an example of the flow when adding a field to work with the ILM
 	    - IAM - Sync User
 	    
 	    You can add the following incident types if you would like to display the new field in the incident layout (the new fields will be shown regardless in the User Profile indicator).
+	    - IAM - Sync User
```

Comment:
did you mean "AD activation" here (since it's already in line 219)?

Reply:
No since AD activation has no layout. Actually I think that since Workday detects the incident type automatically, it's now mandatory to add the field to all the incident types that I wrote, right?
	    - IAM - Sync User
	    - IAM - New Hire
	    - IAM - Update User
	    - IAM - Terminate User
	    - IAM - Rehire User

Reply:
I don't think it's mandatory, because the fields are mapped already in the integration and then passed to the incident with the UserProfile incident field, so they will be added to the indicator. but if the user wants to see the incident field in the layout they will need to associate it to the rest of the types.

Reply:
Discussed on Slack, the integration will utilize the `IAM - Sync User` mapper for all the other incident types (they are using the same fields)